charts/stable/openvscode-server/questions.yaml

```diff
@@ -7,7 +7,6 @@ portals:
       - "$kubernetes-resource_configmap_portal_host"
     ports:
       - "$kubernetes-resource_configmap_portal_port"
-    path: "/?tkn={{ .Values.env.CONNECTION_TOKEN }}"
```

Comment:
questions.yaml CANNOT accept variables in the iX-Systems parts!
This is not used by Helm, it's only used by SCALE.